src/index.ts

```diff
@@ -1,8 +1,12 @@
 import InMemoryArticles from './adaptors/in-memory-articles';
 import createApp from './app';
+import createRouter from './router';
+import Routes from './routes';
 
 const articles = new InMemoryArticles();
+const router = createRouter();
+const apiDocumentationPath = router.url(Routes.ApiDocumentation);
 
-const app = createApp(articles);
+const app = createApp(articles, router, apiDocumentationPath);
```

Comment:
doesn't look like it can scale to lots of dependencies?

Reply:
Could remove `apiDocumentationPath`, and assume the router has a route named `Routes.ApiDocumentation`...